driving_log_replayer/scripts/perception_2d_evaluator_node.py

```diff
@@ -101,25 +92,37 @@ class Perception2DResult(ResultBase):
     def __init__(self, condition: Dict):
         super().__init__()
         self.__pass_rate = condition["PassRate"]
-        self.__success = 0
-        self.__total = 0
+        self.__target_cameras = condition["TargetCameras"]
+        self.__success = {}
+        self.__total = {}
+        self.__result = {}
+        self.__msg = {}
+        for camera_type, _ in self.__target_cameras.items():
```

Comment:
[NITS] It looks enough to use `keys()`
```suggestion
        for camera_type in self.__target_cameras.keys():
```